apps/dialtone-documentation/docs/about/whats-new/posts/2023-12-28.md

```diff
@@ -0,0 +1,68 @@
+---
+heading: Dialtone 9 released and published with @next tag
+author: Tico Ortega
+posted: '2023-12-28'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+Hi everyone, hope you're having a great holidays. As the year comes to an end, it's time for a big update!.
+In this blog post we'll talk about the release of Dialtone 9.
+
+## TLDR; Dialtone 9 released and published with @next tag
+
+We decided to release a mono-package containing all dialtone packages under the name of `@dialpad/dialtone`.
+
+This aligns better with our monorepo approach, helps up to ease the task of maintaining all our packages up-to-date
+on clients using dialtone and makes using multiple dialtone packages easier.
+
+In order to do so, we increased the version of `@dialpad/dialtone` as this is a breaking change but released to
+@next tag to avoid unintentionally updating clients to this version of dialtone yet.
+
+Also, we're going to continue releasing Dialtone CSS library to `@dialpad/dialtone-css` (formerly dialtone) as a
+separate
+package in case someone still needs to use it.
+
+As soon as we can start, we're going to do the migration on product to update the dialtone version and migrate
+the usage of multiple dialtone packages to the mono-package with its multiple entries.
```

Comment:
Hmm maybe just remove this. We'll probably do the migration on ubervoice but not any other projects. I can just include this in the message linking to this post when I send it out.

Reply:
Makes sense, removing